src/index.js

```diff
@@ -320,7 +320,7 @@ function getElementsByTagNameNS (namespaceURI, tagName, context = document) {
 }
 
 function querySelector (selector, context = document) {
-  return query(selector, context, false)
+  return query(selector, context, false) || null
```

Comment:
Guess this should technically be in another PR since we're fixing querySelector, but I think it's fine to put it in this PR

Reply:
D'oh, just realized this is not needed. `querySelector()` was correctly returning null all along